test/mpi/configure.ac

```diff
@@ -186,6 +186,11 @@ AC_ARG_ENABLE(profile-tests,
 		[Include f90 for PMPI (default)])],,
 	[enable_profile_tests=yes])
 
+AC_ARG_ENABLE(f90-sizeof-tests,
```

Comment:
@ggouaillardet what's the reason these tests might not build? I'm mostly just curious, but it would be good to document in the commit message or elsewhere.

Reply:
@raffenet I rebased the PR vs the latest master in order to fix a (trivial) conflict.

I updated the commit message to explain the rationale for this option

> MPI_Sizeof() requires some level of support by the Fortran compiler.
> For example, Open MPI does not provide this subroutine when built
> on a RHEL 7 box with the stock gfortran (4.8.5) compiler.

It would probably best to remove this option and have `configure` automagically figure out if `MPI_Sizeof()` (and possibly the `ft` and `profile` tests too) can be used.

Do you wish me moving forward and implement this ?